.all-contributorsrc

```diff
@@ -57,6 +57,17 @@
       "contributions": [
         "code"
       ]
+    },
+    {
+      "login": "tim-mcgee",
+      "name": "Tim McGee",
+      "avatar_url": "https://avatars.githubusercontent.com/u/14267457?v=3",
+      "profile": "https://github.com/tim-mcgee",
+      "contributions": [
+        "code",
+        "doc",
+        "test"
```

Comment:
Looks like this contribution is only `doc`, could you remove `code` and `test`? (Or contribute code/tests) :-)